samples/flashcards/README.md

```diff
@@ -0,0 +1,102 @@
+![Custom Connectors](https://img.shields.io/badge/Custom%20Connectors-Not%20Required-green.svg "Does not use custom connectors")
+
+## Authors
+
+Solution|Author(s)
+--------|---------
+Flashcards | [Anand Vijayaragavan](https://github.com/anandragav)
+
+## Version history
+
+Version|Date|Comments
+-------|----|--------
+1.0|May 11, 2023|Initial release
+
+## Features
+
+This PowerApp provides a flashcards template to learn any topic with Flashcards.
+
+* Topic: Two sample topics in JSON format included in the App start
+* Animations: Animations for the flashcards included in the app.
+
+## Prerequisites
+
+None
+
+## Data Sources
+
+None
+
+## Minimal Path to Awesome
+
+* [Download](./solution/flashcards.msapp) the `.msapp` from the `solution` folder
+* Use the `.msapp` file using **File** > **Open** > **Browse** within Power Apps Studio.
+* Save and Publish
+
+## Using the Source Code
+
+You can also use the [Power Apps CLI](https://docs.microsoft.com/powerapps/developer/data-platform/powerapps-cli) to pack the source code by following these steps::
+
+* Clone the repository to a local drive
+* Pack the source files back into `.msapp` file:
+
+  ```bash
+  pac canvas pack --sources pathtosourcefolder --msapp pathtomsapp
+  ```
+
+  Making sure to replace `pathtosourcefolder` to point to the path to this sample's `sourcecode` folder, and `pathtomsapp` to point to the path of this solution's `.msapp` file (located under the `solution` folder)
+* Use the `.msapp` file using **File** > **Open** > **Browse** in Power Apps Studio.
+
+## Disclaimer
+
+**THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.**
+
+## Support
+
+While we don't support samples, if you encounter any issues while using this sample, you can [create a new issue](https://github.com/pnp/powerapps-samples/issues/new?assignees=&labels=Needs%3A+Triage+%3Amag%3A%2Ctype%3Abug-suspected&template=bug-report.yml&sample=expandable-navigation&authors=@luisefreese&title=expandable-navigation%20-%20).
+
+For questions regarding this sample, [create a new question](https://github.com/pnp/powerapps-samples/issues/new?assignees=&labels=Needs%3A+Triage+%3Amag%3A%2Ctype%3Abug-suspected&template=question.yml&sample=expandable-navigation&authors=@luisefreese&title=expandable-navigation%20-%20).
+
+Finally, if you have an idea for improvement, [make a suggestion](https://github.com/pnp/powerapps-samples/issues/new?assignees=&labels=Needs%3A+Triage+%3Amag%3A%2Ctype%3Abug-suspected&template=suggestion.yml&sample=expandable-navigation&authors=@luisefreese&title=expandable-navigation%20-%20).
+
+## For more information
+
+* [Companion blog post about this sample](https://www.m365princess.com/blogs/build-powerapps-progressbar-component/)
```

Comment:
The companion blog post must be removed. Has been a copy paste error on my part.